src/frontend/services/wishlistService.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+
+const getWishlist = async (myToken) => {
+    let actionSuccess = false;
+    let actionResponse;
+    await axios.get('/api/user/wishlist', { headers: { authorization: myToken }}).then((response) => {
+      actionResponse = response;
+      actionSuccess = true;
+    }).catch((error) => {
+        actionResponse = error;
+        actionSuccess = false;
+    });
+    return {actionResponse , actionSuccess};
```

Comment:
You can try doing this with try...catch. Would look much cleaner

Reply:
> Suggestion: You can try doing this with try...catch. Would look much cleaner

thanks for the suggestion